common/source/docs/common-oem-customizations.rst

```diff
@@ -6,53 +6,63 @@ OEM Customization
+        include ../CubeOrange/hwdef.dat
 
-.. warning:: The ``defaults.parm`` file should be as small as possible. Some boards only allow 1024 bytes total for this file. Every ASCII byte in the file counts against this limit (except for comment lines). Use integer values where possible. Below is a simple example.
+#. The firmware name can be customized by adding one line to the ``hwdef.dat`` file.
 
-::
+    ::
 
-       # setup for NTF LEDs on output5
-       SERVO5_FUNCTION 120
-       NTF_LED_TYPES 256
+        define AP_CUSTOM_FIRMWARE_STRING "MyMagicFrame"
 
-5. You can also embed :ref:`Lua scripts<common-lua-scripts>` in the ROM of the chip that will automatically run. Since Lua is currently only run on autopilots with a lot of flash space, they are only restricted in total aggregate size to available free flash memory. Put the scripts in a sub-directory called ``scripts``, i.e. libraries/AP_HAL_ChibiOS/hwdef/OEM_CubeOrange/scripts. Files must end in ``.lua``.
+    .. note:: Custom frame type strings can be created through Lua the scripting method ``motors:set_frame_string("Custom frame name")``.
 
-.. warning:: the user may also run Lua scripts off the SD card, so care should be taken in naming the embedded script file names to not conflict with potential user file names. It is recommended that the file names of embedded Lua scripts be provided in the product documentation for the user.
+#. Now in that same directory, copy the base board's ``hwdef_bl.dat`` bootloader file, and then include a file named ``defaults.parm``. This file will be the parameter overrides of the standard defaults to match your system's configuration. Things like output function assignments, auxiliary RC switches, flight and tuning parameters, etc.
 
-1. Now build as normal with OEM-CubeOrange as the board name in the configuration. The default parameters and Lua scripts will be embedded appropriately.
+    .. warning:: The ``defaults.parm`` file should be as small as possible. Some boards only allow 1024 bytes total for this file. Every ASCII byte in the file counts against this limit (except for comment lines). Use integer values where possible. Below is a simple example.
+
+    ::
+
+        # setup for NTF LEDs on output5
+        SERVO5_FUNCTION 120
+        NTF_LED_TYPES 256
+
+#. You can also embed :ref:`Lua scripts<common-lua-scripts>` in the ROM of the chip that will automatically run. Since Lua is currently only run on autopilots with a lot of flash space, they are only restricted in total aggregate size to available free flash memory. Put the scripts in a sub-directory called ``scripts``, i.e. ``libraries/AP_HAL_ChibiOS/hwdef/OEM_CubeOrange/scripts``. Files must end in ``.lua``.
```

Comment:
a bit of duplication from below, but should be here also
```suggestion
#. You can also embed :ref:`Lua scripts<common-lua-scripts>` in the ROM of the chip that will automatically run. Since Lua is currently only run on autopilots with a lot of flash space, they are only restricted in total aggregate size to available free flash memory. Put the scripts in a sub-directory called ``scripts``, i.e. ``libraries/AP_HAL_ChibiOS/hwdef/OEM_CubeOrange/scripts``. Files must end in ``.lua``. You can also have other sub-folders for informational files, board pictures, etc. and these will be included and viewable when examining the @ROMFS folder with MAVFtp.
```

Reply:
I'll split that out into another step and mention that space is limited. Which board spec is it is taken up? Flash?

Reply:
Fixed